utils.py

```diff
@@ -272,7 +272,14 @@ def convert_png_binary_to_data_url(content):
 
 
 def convert_png_to_data_url(filepath):
-    """Converts the png file at filepath to a data URL."""
+    """Converts the png file at filepath to a data URL.
+    
+    Arg:
+        filepath: str. Path of the file.
+    
+    Return:
+        str. Converting png file to data URL.
```

Comment:
Make it `Data URL created from the binary content of the PNG.`

Reply:
done